Error when try execute the image

docker run -it \
    --name Proxy-ADempiere-API \
        -p 8085:8085 \
        erpya/proxy-adempiere-api
yarn install v1.22.5
[1/5] Validating package.json...
[2/5] Resolving packages...
success Already up-to-date.
Done in 1.92s.
yarn run v1.22.5
$ npm run -s build
$ pm2 start ecosystem.json $PM2_ARGS

                        -------------

__/\\\\\\\\\\\\\____/\\\\____________/\\\\____/\\\\\\\\\_____
 _\/\\\/////////\\\_\/\\\\\\________/\\\\\\__/\\\///////\\\___
  _\/\\\_______\/\\\_\/\\\//\\\____/\\\//\\\_\///______\//\\\__
   _\/\\\\\\\\\\\\\/__\/\\\\///\\\/\\\/_\/\\\___________/\\\/___
    _\/\\\/////////____\/\\\__\///\\\/___\/\\\________/\\\//_____
     _\/\\\_____________\/\\\____\///_____\/\\\_____/\\\//________
      _\/\\\_____________\/\\\_____________\/\\\___/\\\/___________
       _\/\\\_____________\/\\\_____________\/\\\__/\\\\\\\\\\\\\\\_
        _\///______________\///______________\///__\///////////////__


                          Runtime Edition

        PM2 is a Production Process Manager for Node.js applications
                     with a built-in Load Balancer.

                Start and Daemonize any application:
                $ pm2 start app.js

                Load Balance 4 instances of api.js:
                $ pm2 start api.js -i 4

                Monitor in production:
                $ pm2 monitor

                Make pm2 auto-boot at server restart:
                $ pm2 startup

                To go further checkout:
                http://pm2.io/


                        -------------

[PM2] Spawning PM2 daemon with pm2_home=/root/.pm2
[PM2] PM2 Successfully daemonized
[PM2][WARN] Applications api, o2m not running, starting...
[PM2][ERROR] script not found : /var/www/proxy-adempiere-api/dist/src/modules/default-vsf/worker/order_to_magento2.js
[PM2] App [api] launched (1 instances)
┌──────────┬────┬────────────┬──────┬─────┬────────┬─────────┬────────┬─────┬───────────┬──────┬──────────┐
│ App name │ id │ version    │ mode │ pid │ status │ restart │ uptime │ cpu │ mem       │ user │ watching │
├──────────┼────┼────────────┼──────┼─────┼────────┼─────────┼────────┼─────┼───────────┼──────┼──────────┤
│ api      │ 0  │ 1.0.0-rc.3 │ fork │ 175 │ online │ 0       │ 0s     │ 0%  │ 25.4 MB   │ root │ disabled │
└──────────┴────┴────────────┴──────┴─────┴────────┴─────────┴────────┴─────┴───────────┴──────┴──────────┘
 Use `pm2 show <id|name>` to get more details about an app
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

--- a/Dockerfile
+++ b/Dockerfile
@@ -30,6 +30,7 @@ RUN mkdir -p /var/www/ && \
 	mv $BINARY_NAME $REPO_NAME && \
 	rm $BINARY_NAME.zip && \
 	cd $REPO_NAME && \
+	cp -rf ./packages/default-vsf ./src/modules && \
 	cp ./docker/proxy-api/proxy-api.sh /usr/local/bin/ && \
 	yarn install --no-cache && \
 	apk del .build-deps

--- a/README.md
+++ b/README.md
@@ -34,5 +34,5 @@ Run the latest container with:
 docker run -it \
     --name Proxy-ADempiere-API \
 	-p 8085:8085 \
-	erpya/proxy-adempiere-adempiere-api
+	erpya/proxy-adempiere-api
 ```

